webextensions/background/module/index.js

```diff
@@ -0,0 +1,10 @@
+import {
+  refreshContextMenuItems,
+  contextMenuClickListener,
```

Comment:
There is no need to define these functions as global functions, and I think they should be enclosed under a namespace like: `contextMenu.refreshItems` and `contextMenu.onClick`.

Reply:
> There is no need to define these functions as global functions, and I think they should be enclosed under a namespace like: contextMenu.refreshItems and contextMenu.onClick.

To not change usages in `background/background.js`, I didn't assign to namespace object.
Should we change them?